vivisect/impemu/platarch/arm.py

```diff
@@ -123,6 +125,9 @@ def _prep(self, funcva, tmode=None):
                     elif armop.mnem == 'ldr':
                         armthumb += 2
 
+                    if armop.prefixes: # highly unlikely for a function to start with a conditional
```

Comment:
still more heuristics.

Reply:
and ARM stores opcode conditional bits in `.prefixes`, a throwback to i386 opcodes